src/Stratis.Bitcoin/P2P/PeerSelector.cs

```diff
@@ -253,7 +252,7 @@ private PeerAddress Random(IEnumerable<PeerAddress> peers)
         /// <inheritdoc/>
         public IEnumerable<PeerAddress> Attempted()
         {
-            return this.peerAddresses.Values.Where(p => p.Attempted && p.ConnectionAttempts <= 10 && p.LastConnectionAttempt < DateTime.UtcNow.AddSeconds(-60));
+            return this.peerAddresses.Values.Where(p => p.Attempted && p.ConnectionAttempts < 10 && p.LastAttempt < DateTime.UtcNow.AddHours(-1));
```

Comment:
ok so the logic as it is proposed in the PR is 
- attempted 10 times every hour
- if the threshold of 12 hours and 5 attempts is reached reset attempts to 0

Should this not be `ConnectionAttempts < PeerAddress.AttemptThreshold`
Then it will be attempt 5 once every hour for 12 hours

Reply:
>    attempted 10 times every hour
    if the threshold of 12 hours and 5 attempts is reached reset attempts to 0

Correct. BUT well spotted, I fixed that if to use the constant.